pubtools/pulplib/_impl/criteria.py

```diff
@@ -219,6 +219,28 @@ def in_(cls, values):
         """
         return InMatcher(values)
 
+    @classmethod
+    def less_than(cls, value):
+        """
+        Returns a matcher for a field whose value is less than the specified input
+        value
+
+        Arguments:
+            values (object)
+                An object to match against the field
+
+        Example:
+            .. code-block:: python
+
+            # would match where last_publish is before "2019-08-27T00:00:00Z
+            # date is reqired to be in this format
+            crit = Criteria.with_field(
+                'last_publish',
+                Matcher.less_than("2019-08-27T00:00:00Z")
```

Comment:
I think if last_publish is stored as a datetime in Pulp/mongo, then we should also be passing a datetime.datetime here.  Can that be made to work?

Reply:
Yes, that's how it will be post the above mentioned change.